tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -0,0 +1,181 @@
+
+```shell
+mycli -u <username> -h <host> -P 4000 -D test --ssl-ca=<CA_root_path> --ssl-verify-server-cert
+```
+
+- Use `--ssl-ca=<CA_root_path>` to set the CA root path on your system.
+- With `--ssl-verify-server-cert` to validate TiDB Serverless Tier clusters.
+
+</div>
+
+<div label="Java">
+
+[MySQL Connector/J](https://dev.mysql.com/doc/connector-j/8.0/en/)'s TLS connection configurations are used here as an example.
+
+```
+jdbc:mysql://<host>:4000/test?user=<username>&password=<your_password>&sslMode=VERIFY_IDENTITY&enabledTLSProtocols=TLSv1.2,TLSv1.3
+```
+
+- Set `sslMode=VERIFY_IDENTITY` to enable TLS and validate TiDB Serverless Tier clusters. JDBC trusts system CA root certificates by default, so you do not need to configure certificates.
+- Set `enabledTLSProtocols=TLSv1.2,TLSv1.3` to restrict the versions of TLS protocol.
+
+</div>
+
+<div label="Python">
+
+[mysqlclient](https://pypi.org/project/mysqlclient/)'s TLS connection configurations are used here as an example.
+
+```
+host="<host>", user="<username>", password="<your_password>", port=4000, database="test", ssl_mode="VERIFY_IDENTITY", ssl={"ca": "<CA_root_path>"}
+```
+
+- Set `ssl_mode="VERIFY_IDENTITY"` to enable TLS and validate TiDB Serverless Tier clusters.
+- Set `ssl={"ca": "<CA_root_path>"}` to set the CA root path on your system.
+
+</div>
+
+<div label="Go">
+
+[Go-MySQL-Driver](https://github.com/go-sql-driver/mysql)'s TLS connection configurations are used here as an example.
+
+```
+mysql.RegisterTLSConfig("tidb", &tls.Config{
+  MinVersion: tls.VersionTLS12,
+  ServerName: "<host>",
+})
+
+db, err := sql.Open("mysql", "<usename>:<your_password>@tcp(<host>:4000)/test?tls=tidb")
+```
+
+- Register `tls.Config` in connection to enable TLS and validate TiDB Serverless Tier clusters. Go-MySQL-Driver uses system CA root certificates by default, so you do not need to configure certificates.
+- Set `MinVersion: tls.VersionTLS12` to restrict the versions of TLS protocol.
+- Set `ServerName: "<host>"` to verify TiDB Serverless Tier's hostname.
+- If you do not want to register a new TLS configuration, you can just set `tls=true` in the connection string.
+
+</div>
+
+<div label="Node.js">
+
+[Mysql2](https://www.npmjs.com/package/mysql2)'s TLS connection configurations are used here as an example.
+
+```
+host: '<host>', port: 4000,user: '<username>', password: '<your_password>', database: 'test', ssl: {minVersion: 'TLSv1.2', rejectUnauthorized: true}
+```
+
+- Set `ssl: {minVersion: 'TLSv1.2'}` to restrict the versions of TLS protocol. 
+- Set `ssl: {rejectUnauthorized: true}` to validate TiDB Serverless Tier clusters. Mysql2 uses system CA root certificates by default, so you do not need to configure certificates.
+
+</div>
+</SimpleTab>
+
+## Where is the CA root path on my system?
+
+The following lists the CA root paths on common platforms.
+
+<SimpleTab>
```

Comment:
The PDF of the content will look like this, which is hard to tell the platform information. 
![image](https://user-images.githubusercontent.com/79440533/198171510-4d2ef770-c44f-4171-a262-b78f76ee496b.png)

Could we use a list instead of a simple tab for this case? 

For example:
- macOS: `/etc/ssl/cert.pem`
- Debian, Ubuntu, or Arch: `/etc/ssl/certs/ca-certificates.crt`

Reply:
I thought we're going to remove PDF for cloud docs. If this is not going to happen very soon, i can remove simple tabs